UI/src/view/linear_asset/laneModellingStyle.js

```diff
@@ -62,10 +62,21 @@
       return me.lineFeatures(me.getNewFeatureProperties(linearAssets, laneNumber));
     };
 
+    var numberOfLanes = function (asset) {
```

Comment:
var numberOfAdditionalLanes seems more recognizable since you are counting only that

Reply:
Done